sdk/multiclients/modular-multiclients/review/multiclient-bar-sub-hlc.api.md

```diff
@@ -0,0 +1,69 @@
+## API Report File for "@msinternal/multiclient"
```

Comment:
the file name reflects what the main entry point is when generated this api view markdown file.  For example bar-sub-hlc means Bar High Level Sub Client.